api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -44,59 +46,82 @@
-  private Timestamp demographicSurveyCompletionTime;
   private boolean disabled;
-  private Short emailVerificationStatus;
   private Set<DbPageVisit> pageVisits = new HashSet<>();
-  private String aboutYou;
-  private String areaOfResearch;
-  private Integer billingProjectRetries;
-  private Integer moodleId;
+  private Timestamp demographicSurveyCompletionTime;
   private Timestamp creationTime;
   private Timestamp lastModifiedTime;
+
+  // possibly obsolete system account fields (not used)
+
+  private Short freeTierCreditsLimitDaysOverride = null;
+  private Timestamp lastFreeTierCreditsTimeCheck;
+  private Timestamp firstRegistrationCompletionTime;
+  private Integer billingProjectRetries;
+
+  // user-editable Profile fields
+
+  private String givenName;
+  private String familyName;
+  private String professionalUrl;
+  private List<Short> degrees;
+  private String areaOfResearch;
   private DbDemographicSurvey demographicSurvey;
   private DbAddress address;
 
+  // potentially obsolete profile-style fields (not used)
+
+  private String phoneNumber;
+  private String currentPosition;
+  private String organization;
+  private String aboutYou;
+
   // Access module fields go here. See http://broad.io/aou-access-modules for docs.
+
   private String eraCommonsLinkedNihUsername;
   private Timestamp eraCommonsLinkExpireTime;
   private Timestamp eraCommonsCompletionTime;
+  private Timestamp eraCommonsBypassTime;
+
   private String rasLinkLoginGovUsername;
   private Timestamp rasLinkLoginGovCompletionTime;
   private Timestamp rasLinkLoginGovExpireTime;
   private Timestamp rasLinkLoginGovBypassTime;
+
   private Timestamp betaAccessRequestTime;
   private Timestamp betaAccessBypassTime;
+
   private Timestamp dataUseAgreementCompletionTime;
   private Timestamp dataUseAgreementBypassTime;
   private Integer dataUseAgreementSignedVersion;
+
   private Timestamp complianceTrainingCompletionTime;
   private Timestamp complianceTrainingBypassTime;
   private Timestamp complianceTrainingExpirationTime;
-  private Timestamp eraCommonsBypassTime;
-  private Timestamp emailVerificationCompletionTime;
-  private Timestamp emailVerificationBypassTime;
-  private Timestamp idVerificationCompletionTime;
-  private Timestamp idVerificationBypassTime;
+
   private Timestamp twoFactorAuthCompletionTime;
   private Timestamp twoFactorAuthBypassTime;
+
   private Timestamp profileLastConfirmedTime;
+
   private Timestamp publicationsLastConfirmedTime;
 
+  // potentially obsolete access module fields.  These are likely to be deleted in the near future.
+
+  private Boolean idVerificationIsValid;
+  private Timestamp idVerificationCompletionTime;
+  private Timestamp idVerificationBypassTime;
+
+  private Short emailVerificationStatus;
+  private Timestamp emailVerificationCompletionTime;
+  private Timestamp emailVerificationBypassTime;
+
+  // Moodle badges are indexed by username, not this value.  See ComplianceService.
```

Comment:
I don't recall much about this field. Is the call to remove it, or is it actually being used for something?

Reply:
Remove it.  If this value was ever used, it isn't now.

ComplianceService and MoodleApi have a "Moodle ID" but there does not seem to be any connection with this.